src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/CreateMergedManifestBuildModelTests.cs

```diff
@@ -32,23 +29,81 @@ class CreateMergedManifestBuildModelTests
         private static readonly string buildNumber = "azDevBuildNumber";
         private static readonly string sourceBranch = "thisIsASourceBranch";
         private static readonly string commitSourceVersion = "thisIsASourceVersion";
+        private static readonly string id = "1234";
+        private static readonly string version = "thisIsVersion";
+        private static readonly string noVersion = "thisIsNoVersion";
```

Comment:
What does noVersion mean? it looks like if you're setting version to be noVersion, you're still setting a version? Why do some of the test packages have a null version field, while others set it to noVersion?

Reply:
Removed it and used and actual version, so that it makes more sense in the test.